src/util/mockDataUtils.ts

```diff
@@ -46,11 +46,25 @@ export const fakeEvents = (
 });
 
 export const fakeEvent = (overrides?: Partial<EventDetails>): EventDetails => {
+  const names: string[] = [];
+
+  const uniqueName = (): string => {
+    const name = faker.name.title();
+
+    if (names.includes(name)) {
+      return uniqueName();
+    } else {
+      names.push(name);
+    }
+
+    return name;
+  };
+
   return merge(
     {
       id: `hel:${faker.random.uuid()}`,
       internalId: faker.random.uuid(),
-      name: fakeLocalizedObject(faker.name.title()),
+      name: fakeLocalizedObject(uniqueName()),
```

Comment:
Is there same risk of having duplicates in other tests too?

Reply:
Hmm, maybe? What if we went like this now, and if in the future if we still get this error, then we could make a utility which could compare all the fields and make sure there are no duplicates.